### ISSUE TITLE
Rail connections are not save correctly

### ISSUE THREAD

**Issue by josh-65** [EngineHub/WorldEdit#1973]
### WorldEdit Version

7.2.8

### Platform Version

Paper 1.18.1

### Confirmations

- [X] I am using the most recent Minecraft release.
- [X] I am using a version of WorldEdit compatible with my Minecraft version.
- [X] I am using the latest or recommended version of my platform software.
- [X] I am NOT using a hybrid server, e.g. a server that combines Bukkit and Forge. Examples include Arclight, Mohist, and Cardboard.
- [X] I am NOT using a fork of WorldEdit, such as FastAsyncWorldEdit (FAWE) or AsyncWorldEdit (AWE)

### Bug Description

If i have 2 rails lines ingoing in parallel next to each other I end up getting rails going in all random directions.
![image](https://user-images.githubusercontent.com/26977075/146044932-e0af46cf-6f3e-4c1c-bf14-b48f6012d037.png)

### Expected Behavior

The rails should stay as they were (two straight rows)

### Reproduction Steps

_

**Comment by wizjany** [comment]
duplicate of #1416
